spec/models/file_search_spec.rb

```diff
@@ -31,7 +50,7 @@
     let(:documents) { document_ids.map { |id| double("SolrDocument", id: id) } }
 
     before do
-      allow(catalog_query).to receive(:search_results).with(expected_query, logic) { [double("Response"), documents] }
+      allow(catalog_query).to receive(:search_results).with(expected_query, logic) { [response, documents] }
```

Comment:
This before block is duplicated with the one above.  Can they be pulled out to an outer level?